meshtastic/device_ui.options

```diff
@@ -1,5 +1,7 @@
 *DeviceUIConfig.screen_brightness int_size:8
 *DeviceUIConfig.screen_timeout int_size:16
+*DeviceUIConfig.pin_code int_size:32
```

Comment:
This is already the default size. Did you mean to make it 16?

Reply:
oh, ok .. I'll remove it then